src/components/ui/card.tsx

```diff
@@ -37,32 +37,28 @@ const CardTitle = React.forwardRef<
     ref={ref}
     className={cn(
       "text-2xl font-semibold leading-none tracking-tight",
-      className
+      className,
     )}
     {...props}
   />
-))
-CardTitle.displayName = "CardTitle"
+));
+CardTitle.displayName = "CardTitle";
 
 const CardDescription = React.forwardRef<
   HTMLParagraphElement,
   React.HTMLAttributes<HTMLParagraphElement>
 >(({ className, ...props }, ref) => (
-  <p
-    ref={ref}
-    className={cn("text-sm text-slate-500 dark:text-slate-400", className)}
```

Comment:
I don't think we should remove dark mode, instead change the dark mode strategy.

Reply:
https://tailwindcss.com/docs/dark-mode

Reply:
Hmm...right.  The reason I removed is because dark mode was not implemented website wide (as of now) . Do you think we should implement this now ? If yes, then I will create a ticket for it 👍